portfolio/src/main/webapp/index.html

```diff
@@ -10,54 +10,104 @@
     <div class="all-content" id="content">
         <table id="allTable" class="table-allInfo">
             <tr>
-                <td id="name">David Vargas Puga</td>
-            </tr>
-            <tr>
-                <td id="aboutMe">About Me</td>
+                <td id="avi">
+                    <img id="avatar" class="circular--square" src="./images/avi.jpg">
+                </td>
             </tr>
             <tr>
-                <td bgcolor="#FFA500" id="aboutMe-info">A bilingual and extroverted Computer Science and Spanish double-major with exceptional communication, interpersonal and leadership skills. First-generation student who thrives in cultures that embrace diversity and reward imagination. Experience in software engineering, Android development and machine learning. </td>
+                <td id="name"><h1>David Vargas Puga</h1></td>
             </tr>
+            <tr id="socials-row">
+                    <td>
+                        <ul id="socials">
+                            <li id="github">
+                                <a href="https://github.com/davidvargaspuga"><img class="social-img" id="github-img" src="./images/github_logo_trans.png" /></a>
+                            <li>
+                            <li id="linkedin">
+                                <a href="https://www.linkedin.com/in/david-vargas-puga/"><img class="social-img" id="linkedin-img" src="./images/linkedin_logo.png" /></a>
+                            <li>
+                            <li id="insta">
+                                <a href="https://www.instagram.com/davidvargas.jpg/"><img class="social-img" id="insta-img" src="./images/insta_logo.png" /></a>
+                            <li>
+                        </ul>
+                    </td>
+                </tr>
             <tr>
-                <td id="socials-row">
-                    <ul id="socials">
-                        <li id="github">
-                            <a href="https://github.com/davidvargaspuga"><img id="github-img" src="./images/github_logo_trans.png" /></a>
-                        <li>
-                        <li id="linkedin">
-                            <a href="https://www.linkedin.com/in/david-vargas-puga/"><img id="linkedin-img" src="./images/linkedin_logo.png" /></a>
-                        <li>
-                    </ul>
-                </td>
+                <td id="aboutMe"><h2 class="section-titles">About Me</h2></td>
             </tr>
             <tr>
-                <td id="projects">Projects</td>
+                <td bgcolor="#FFA500" id="aboutMe-info">A bilingual and extroverted Computer Science and Spanish double-major at the University of Portland with exceptional communication, interpersonal and leadership skills. First-generation student who thrives in cultures that embrace diversity and reward imagination. Experience in software engineering, Android development and machine learning. </td>
```

Comment:
Use a CSS class and set #FFA500 to the background-color CSS property instead. We should avoid mixing styling from content as much as possible.

And same comment about ID name for #aboutMe-info as my other comment